### PR TITLE
First commit, moved updateMotors() to periodic

### PR DESCRIPTION
**Pull request by caseyjbrotherton**
# Pull Request Details
Change updateMotors to be in periodic to reduce auton complexity

## Description

UpdateMotors was previously in a command, this makes it part of the periodic function of the armsubsystem

## Which does this affect?
ArmSubsystem
Auton
## Testing Done

Calvin tested, will test further

## Checklist

- [X] My code follows the code style of this project.
- [X] My code has been tested on the Robot OR includes simulation code to prove functionality.

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/ArmSubsystem.java [hunk 1]
```diff
@@ -323,7 +323,7 @@ public void cancelCommands() {
 
 
   public void updateSimMotors() {
-    updateMotors();
+    //updateMotors(); Moved to periodic
```
Comment (caseyjbrotherton) [inline src/main/java/frc/robot/subsystems/ArmSubsystem.java:326]: Don't leave dead code, remove

File: src/main/java/frc/robot/subsystems/ArmSubsystem.java [hunk 2]
```diff
@@ -385,7 +385,8 @@ public Command goToState(ArmState state){
     //Command for autonomous, obstructs routine until arm is at setpoint
     return new FunctionalCommand(
                 () -> setState(state), //Init
-                this::updateMotors, //Execute
+                //this::updateMotors, //Moved to periodic
```
Comment (caseyjbrotherton) [inline src/main/java/frc/robot/subsystems/ArmSubsystem.java:388]: Don't leave dead code, remove

File: src/main/java/frc/robot/subsystems/ArmSubsystem.java [hunk 3]
```diff
@@ -385,7 +385,8 @@ public Command goToState(ArmState state){
...
+                null, //Execute
```
Comment (azvanderpas) [inline src/main/java/frc/robot/subsystems/ArmSubsystem.java:389]:
```suggestion
                () -> {}, //Execute
```
Null doesn't work here

**Comment by azvanderpas** [comment]
Need to also fix auto paths that call updateMotors as workarounds

**Comment by azvanderpas** [comment]
This is redundant with one-score-cube. Closing

**Pull request closed without merging**